packages/nimble-components/src/unit/byte/tests/unit-byte.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { html } from '@microsoft/fast-element';
+import { type Fixture, fixture } from '../../../utilities/tests/fixture';
+import { UnitByte, unitByteTag } from '..';
+import { Byte1024UnitScaleFormatter } from '../models/byte-1024-unit-scale-formatter';
+import { ByteUnitScaleFormatter } from '../models/byte-unit-scale-formatter';
+
+async function setup(): Promise<Fixture<UnitByte>> {
+    return fixture<UnitByte>(html`
+        <nimble-unit-byte binary></nimble-unit-byte>
+    `);
+}
+
+describe('Byte unit', () => {
+    let element: UnitByte;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+        await connect();
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('should export its tag', () => {
+        expect(unitByteTag).toBe('nimble-unit-byte');
+    });
+
+    it('can construct an element instance', () => {
+        expect(document.createElement('nimble-unit-byte')).toBeInstanceOf(
+            UnitByte
+        );
+    });
+
```

Comment:
The 'returns ByteScaleFormatter when "binary" is false' test was _almost_ testing that case, except that I was explicitly setting the `binary` property to `false`. I changed the test setup, so that I'm instead just leaving the `binary` attribute off. Now it covers the case you mentioned.